GUI/datasets.py

```diff
@@ -70,8 +70,8 @@ def get_vocab(self):
         return self._vocab
 
 def _setup_datasets(data, root, ngrams=1, vocab=None, include_unk=False, rebuild=False):
-	train_csv_path = root + 'train.csv'
-	test_csv_path = root + 'test.csv'
+	train_csv_path = 'train.csv'
```

Comment:
Similar comments as above: is there a case where the user could change where the model is supposed to be loaded from?